pyproject.toml

```diff
@@ -21,6 +21,7 @@ dependencies = [
 test = [
     "black[jupyter]",
     "pytest",
+    "pytest-cov",
     "ruff==0.0.238"
 ]
```

Comment:
I use `hatch` CLI locally and these deps are repeated in `[tool.hatch.envs.default]`.  We should add `pytest-cov` there as well I think. 

side note: do you know if it's possible to avoid this duplication? i.e., make `optional-dependencies` dynamic or force hatch to install optional deps in the default envs?

Reply:
yeah, you can avoid it with `features`:
```py
[tool.hatch.envs.default]
features = ["test"]
```

will do here

Reply:
oh, slick!